eng/common/Scripts/MaestroHelpers.ps1

```diff
@@ -54,13 +54,16 @@ function ConvertToMaestroFriendlyAzureDevOpUri([string]$buildRepositoryUri)
 function IsGitHubRepo([string]$buildRepositoryUri)
 {
     $githubUrls = @("https://github.com", "https://wwww.github.com")
```

Comment:
There are 4 w's here "wwww".  Is that intentional?

Reply:
@kythant - Do you Know?

Reply:
This is definitely a typo.